src/cmds/mem/Infomem.my

```diff
@@ -0,0 +1,18 @@
+package embox.cmd.mem
+
+@AutoCmd
+@Cmd(name = "infomem",
+	help = "get mem info",
+	man = '''
+		NAME
+			infomem - get mem info.
+		SYNOPSIS
+			infomem
+		DESCRIPTION
+
+		AUTHOR
+			Evgenij Bogdanov
+	''')
+module infomem {
+	source "infomem.c"
+}
```

Comment:
Probably you need dependencies on what you're using. `depends embox.mem.phymem` etc.

Reply:
Done.